src/Command/AbstractClientRequestCommand.php

```diff
@@ -68,6 +71,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             ->setRaw($input->getOption('raw') !== false)
             ->setDefaultAuthMethod($this->defaultAuthMethod);
 
+        // In case of a dry-run the request to TER is skipped.
+        if ($input->getOption('dry-run') === true) {
+            return 0;
```

Comment:
needs an info in CLI output IMHO

Reply:
Agree, already dicussed this with Oli via PM.